lib/linux_admin/ip_addr.rb

```diff
@@ -0,0 +1,31 @@
+module LinuxAdmin
+  class IpAddr
```

Comment:
@Fryguy If we go the route of creating a larger class we would be able to add stuff like DNS servers to it rather than having a separate class for that, but in that case I would also want to move the functionality from https://github.com/ManageIQ/linux_admin/blob/master/lib/linux_admin/hosts.rb to the larger class.  Is that a change we want to make to the API?